### PR TITLE
Document Azure ServiceBus params for AAD auth

### PR DESCRIPTION
**Pull request by halspang**
## Description

For both the pubsub and input/output bindings using Azure
ServiceBus, antoher parameter is required in order to use AAD auth.
This commit adds documentation for that parameter.

https://github.com/dapr/docs/issues/1867

Signed-off-by: Hal Spang <halspang@microsoft.com>

## Issue reference

Closes: #1867

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md [hunk 1]
```diff
@@ -42,6 +67,7 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | connectionString | Y | Input/Output | The Service Bus connection string | `"Endpoint=sb://************"` |
 | queueName | Y | Input/Output | The Service Bus queue name. Queue names are case-insensitive and will always be forced to lowercase. | `"queuename"` |
 | ttlInSeconds | N | Output | Parameter to set the default message [time to live](https://docs.microsoft.com/azure/service-bus-messaging/message-expiration). If this parameter is omitted, messages will expire after 14 days. See [also](#specifying-a-ttl-per-message) | `"60"` |
+| namespaceName| N | Input/Output | Parameter to set the name of the service bus namespace. Required if using AAD authentication. | `"namespace"` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md]:
```suggestion
| namespaceName| N | Input/Output | Parameter to set the name of the Service Bus namespace. Required if using AAD authentication. | `"namespace"` |
```

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md [hunk 2]
```diff
@@ -83,6 +107,7 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | connectionRecoveryInSec | N  |Time in seconds to wait between connection recovery attempts. Defaults: `2` | `2`
 | publishMaxRetries | N  | The max number of retries for when Azure Service Bus responds with "too busy" in order to throttle messages. Defaults: `5` | `5`
 | publishInitialRetryInternalInMs | N  | Time in milliseconds for the initial exponential backoff when Azure Service Bus throttle messages. Defaults: `500` | `500`
+| namespaceName| N | Parameter to set the name of the service bus namespace. Required if using AAD authentication. | `"namespace"` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md]:
```suggestion
| namespaceName| N | Parameter to set the name of the Service Bus namespace. Required if using AAD authentication. | `"namespace"` |
```

**Comment by msfussell** [comment]
@halspang  - This is looking good, but we need to add the AAD info into the table. I suggest that you take the same approach that Berndt did with this PR for EventHubs https://github.com/dapr/docs/pull/2327/files which you can see in the docs here https://v1-7.docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-eventhubs/ 

Notable this is being clear that AAD and Connection string are mutually exclusive. 

I like the separate example spec that you have provided also.

**Comment by halspang** [comment]
@msfussell - Are you referring to the `Azure Active Directory (AAD) authentication` in that doc? That's already in this one, it was added when we added the AAD auth, we just forgot the extra parameter that this doc PR adds. Or am I missing something else?

**Comment by msfussell** [comment]
These look great! I did one small update where there was a duplicate header

**Review by msfussell — APPROVED** [review]
Updates look great

**Pull request merged**